blip-tap.md

```diff
@@ -919,149 +919,251 @@ order to clamp down on price volatility exposure.
+| 1        | rfq_id                    | 32*byte    |
+| 2        | transfer_type             | 1*byte     |
+| 3        | expiry                    | 8*byte     |
+| 4        | in_asset_id               | 32*byte    |
+| 5        | in_asset_group_key        | 33*byte    |
+| 6        | out_asset_id              | 32*byte    |
+| 7        | out_asset_group_key       | 33*byte    |
+| 8        | in_asset_to_btc           | FixedPoint |
+| 9        | out_asset_to_btc          | FixedPoint |
+| 10       | max_transferable_in_asset | BigSize    |
 
 where:
 
-* `rfq_id` is a randomly generate 32-byte value to uniquely identify this RFQ
-  request
-* `asset_id` is the asset ID of the asset the receiver intends to receive
-* `asset_amt` is the amount of units of said asset
-* `suggested_rate_tick` is the internal unit used for asset conversions. A tick
-  is 1/10000th of a currency unit. It gives us up to 4 decimal places of
-  precision (0.0001 or 0.01% or 1 bps). As an example, if the BTC/USD rate was
-  $61,234.95, then we multiply that by 10,000 to arrive at the `usd_rate_tick`:
-  `$61,234.95 * 10000 = 612,349,500`.  To convert back to our normal rate, we
-  decide by `10,000` to arrive back at `$61,234.95`.
-
-Given valid `rfq_id`, we then define an `tap_rfq_scid` by taking the last `8`
-bytes of the `rfq_id` and interpreting them as a 64-bit integer. 
+* `version` is the version of the quote request message; the current version
+  is `0`.
+* `rfq_id` is a randomly generated 32-byte value to uniquely identify this RFQ
+  request.
+* `transfer_type` (`uint8`): Specifies the type of transaction that will take
+  place if the quote request results in an accepted agreement. It is set to:
+    * `0` when the transfer type is unspecified.
+    * `1` when the requesting peer is attempting to pay a lightning invoice.
+    * `2` when the requesting peer is seeking to receive funds corresponding to
+      a lightning invoice.
+* `expiry` is the Unix timestamp in seconds representing the exact time after
+  which the quote request and rate information are no longer valid.
+* `in_asset_id` represents the identifier of the asset which will be inbound
+  to the requesting peer (therefore outbound to the counterparty peer); an
+  asset ID of all zeros indicates BTC.
+* `in_asset_group_key` is the serialized compressed public group key of the
+  asset which will be inbound to the requesting peer (therefore outbound to
+  the counterparty).
+* `out_asset_id` represents the identifier of the asset which will be outbound
+  to the requesting peer (and therefore inbound to the counterparty); an asset
+  ID of all zeros indicates BTC.
+* `out_asset_group_key` is the serialized compressed public group key of the
+  asset which will be outbound to the requesting peer (and therefore inbound to
+  the counterparty).
+* `in_asset_to_btc` is the proposed in-asset to BTC conversion rate represented
+  as a [fixed-point number](#fixed-point-number-type).
```

Comment:
Should mention the static value of 100 billion milli-satoshi when this is BTC. Same for all `in/out_asset_to_btc` in this doc. Or can generally mention it in the `#fixed-point-number-type` section.

Reply:
Sure, we could leave this empty if we wanted to. But since the description says "in-asset to BTC", then this could be interpreted as 1 if the "asset" is BTC. But actually the asset is milli-satoshi, so the explicit value makes it more obvious. Also, it would mean we don't need any logic around this and can just pass along the value from the oracle.

Reply:
I think I've managed to make this clear in the BLIP now.